webrtc.html

```diff
@@ -3767,8 +3764,11 @@ <h2>
                             </li>
                           </ol>
                           <p>
-                            The filtering MUST NOT change the order of the
-                            codec preferences.
+                            The <i>codec preferences</i> of a [= media
+                            description =]'s [= associated =] transceiver,
+                            <var>transceiver</var>, is said to be the value of
+                            <var>filteredCodecs</var> if non-empty and said to
+                            be unset otherwise.
```

Comment:
Note to self there is also a reference to preferred codecs internal slots in setParameters that should be filtered on sending